plugins/modules/docker_swarm_service.py

```diff
@@ -344,6 +345,12 @@
           - Requires API version >= 1.27.
         type: list
         elements: dict
+      replicas_max_per_node:
+        description:
+          - Maximum number of tasks per node.
+          - Corresponds to the C(--replicas_max_per_node) option of C(docker service create).
+          - Requires API version >= 1.40
+        type: int
```

Comment:
```suggestion
        type: int
        version_added: 1.3.0
```

Reply:
Let's see if this is the problem. Iinteresting that only tests on two versions of RHEL failed.

Reply:
The tests probably only run on that two nodes. (All swarm-related tests don't run in the docker containers, only on VMs.)